### DISCUSSION TITLE
Exceeding maximum Canvas Limit

### DISCUSSION THREAD

**Discussion by MrRenesto** [cytoscape/cytoscape.js#3253] [Q&A]
Hello,

i have a problem Exceeding the Canvas Limit on Firefox of 32,767 pixels

ERROR DOMException: CanvasRenderingContext2D.setTransform: Canvas exceeds max size.
    setContextTransform cytoscape.esm.mjs:30125
    render cytoscape.esm.mjs:30209
    renderFn2 cytoscape.esm.mjs:26717

Didnt find anything regarding this problem. Can Cytoscape handle this or do i have to make a 2nd Canvas if i want to continue expanding  my Graph ?

Appreciate any help :)

**Comment by maxkfranz** [comment]
Browser limitation.

Use a smaller pixel density/ratio if you want to support very large displays.  Most browsers and devices are fine out of the box.
